signer/remote/remote_test.go

```diff
@@ -1,8 +1,7 @@
-package remote
+package remote_test
```

Comment:
NewHandlerFromSigner doesn't have to live in api/sign/sign.go, maybe there's a better package to put it in that doesn't require signer/universal